airbyte-integrations/connectors/source-twilio-taskrouter/source_twilio_taskrouter/spec.yaml

```diff
@@ -0,0 +1,25 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/twilio-taskrouter
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Twilio Taskrouter Spec
+  type: object
+  required:
+    - account_sid
+    - auth_token
+  additionalProperties: true
+  properties:
+    # 'TODO: This schema defines the configuration required for the source. This usually involves metadata such as database and/or authentication information.':
+    account_sid:
+      type: string
+      description: Twilio Account ID
+      airbyte_secret: true
+    auth_token:
+      type: string
+      description: Twilio Auth Token
+      airbyte_secret: true
+    sid:
```

Comment:
is it better if it's renamed to workspaceId? 

If it's an optional property, and it's empty, the workers stream would fail right? can you update the documentation to reflect that?

Reply:
Sure, I'll replace sid with workspaceId. I just need to indicate in the docs that the user must enter a workspaceId for the workers stream to work and point to the docs where to find it, that should be enough i guess.